R-package/tests/testthat/test_basic.R

```diff
@@ -51,6 +51,7 @@ test_that("use of multiple eval metrics works", {
 
 
 test_that("training continuation works", {
+  testthat::skip("This test is currently broken. See #2468 for details.")
```

Comment:
Just to make it a bit more obvious (for any user), change to `This test is currently broken. See issue #2468 for details.`

But if it's only for us then there's no need to change.

Reply:
oh sure! I can add that, good idea